.github/workflows/e2e-ci.yaml

```diff
@@ -11,6 +11,9 @@ on: # rebuild any PRs and main branch changes
       TestsToRun:
         default: '[\"SensorDecodingTest\",\"OTAAJoinTest\",\"ABPTest\",\"OTAATest\",\"MacTest\",\"ClassCTest\",\"C2DMessageTest\",\"MultiGatewayTest\"]'
         description: 'tests to run'
+      TxPower:
+        description: 'TXPower value to use in E2E tests'
+        default: '14'
```

Comment:
could we put the default to something lower, like 6?

Reply:
Done in c7aec0970a423076b6830852326ec72924e399f5